module/VuFind/tests/integration-tests/src/VuFindTest/Mink/CallnumberBrowseTest.php

```diff
@@ -147,19 +147,21 @@ protected function activateAndTestLinks($type, $page, $expectLinks)
                 ]
             ]
         );
-        $this->getMinkSession()->reload();
-        $this->snooze();
-        $link = $page->find('css', '.callnumber a,.groupCallnumber a,.fullCallnumber a');
+        $session = $this->getMinkSession();
+        $session->reload();
+        $this->waitForPageLoad($page);
+        $link = $this->findCss($page, '.callnumber a,.groupCallnumber a,.fullCallnumber a');
         $this->checkLink($link, $type);
 
         // Multiple callnumbers
         $this->setupMultipleCallnumbers();
-        $this->getMinkSession()->reload();
-        $this->snooze();
-        $link = $page->find('css', '.callnumber a,.groupCallnumber a,.fullCallnumber a');
+        $session->reload();
+        $this->waitForPageLoad($page);
         if ($expectLinks) {
+            $link = $this->findCss($page, '.callnumber a,.groupCallnumber a,.fullCallnumber a');
             $this->checkLink($link, $type);
         } else {
+            $link = $page->find('css', '.callnumber a,.groupCallnumber a,.fullCallnumber a');
             $this->assertTrue(null === $link);
```

Comment:
We could simplify this to an `assertNull`.

Reply:
Done.